services/datamanager/builtin/capture_test.go

```diff
@@ -276,10 +283,9 @@ func passTime(ctx context.Context, mc *clk.Mock, interval time.Duration) chan st
 }
 
 // testFilesContainSensorData verifies that the files in `dir` contain sensor data, and that there are `expectedUniqueDataValues` num of unique datapoints.
-func testFilesContainSensorData(t *testing.T, dir string, expectedUniqueDataValues int) {
+func testFilesContainSensorData(t *testing.T, dir string, validateSensorData func(*testing.T, []*v1.SensorData)) {
```

Comment:
[nit]: Small request to update comment to reflect change in test structure

Reply:
Agh thanks for the reminder! Done